src/Server/ServerHelpers.ts

```diff
@@ -188,6 +188,14 @@ export function processSingleServerGrowth(server: Server, threads: number, cores
     usedCycles = Math.min(Math.max(0, Math.ceil(usedCycles)), threads);
     server.fortify(2 * ServerConstants.ServerFortifyAmount * usedCycles);
   }
+  // Prevent returning NaN. This happens when server.moneyMax is 0.
+  if (server.moneyAvailable === 0 && oldMoneyAvailable === 0) {
+    return 0;
```

Comment:
Why not return 1 here? It's arguably at least as sensible as 0 ("0/0, cancel the 0s to get 1"), and it avoids needing a ternary everywhere you use this (can just subtract 1 and log).